src/js/modal-download.js

```diff
@@ -158,6 +158,11 @@ function createElement (item) {
 		item.embargoMessage = item.embargoPeriod ? interpolate(MESSAGES.EMBARGO, item) : '';
 	}
 
+	const hasTranslationComponent = document.getElementById('ftlabsTranslationContainer');
+	if(!!hasTranslationComponent) {
+		item.translationMessage = item ? interpolate(MESSAGES.ENGLISH, item) : '';
```

Comment:
If `item` is falsy, then won't this break anyway when trying to assign `translationMessage` with `TypeError: Cannot set property 'translationMessage' of undefined`?

Should this at least be just
```js
item.translationMessage = interpolate(MESSAGES.ENGLISH, item);
```


Also, it doesn't seem like the `translationMessage` property is ever being used, so how is this showing up in the modal?

Reply:
translationMessage is used in the messages in config, as modelled with the embargo case. Although you're right, it's a mistake and shouldn't check for item being false but `hasTranslation`